packages/alfa-rules/src/sia-r62/serialise.ts

```diff
@@ -71,6 +71,33 @@ export namespace Serialise {
     );
   }
 
+  export function font(style: Style): string {
+    const optional = (["style", "weight"] as const)
+      .map((property) => getLonghand(style, `font-${property}`))
+      .join(" ");
+
+    const size = style.computed("font-size");
+    const family = style.computed("font-family");
+
+    if (optional !== " ") {
+      // Optional properties were changed, need to output the mandatory ones.
+      return normalize(`${optional} ${size} ${family}`);
+    }
+
+    if (
+      size.value.equals(Property.get("font-size").initial.value) &&
+      family.value.equals(Property.get("font-family").initial.values[0])
```

Comment:
We should only get the first in the list of the computed one. So, I  guess,
```suggestion
      family.value.values[0].equals(Property.get("font-family").initial.values[0])
```
```suggestion
      family.value.equals(Property.get("font-family").initial.values[0])
```